guides/release/accessibility/components.md

```diff
@@ -6,7 +6,7 @@ Browsers have implemented the spec in a way that provides functionality for free
 For example, consider this code sample: 
 
 ```html
-<button>Submit Form</button>
+<button type="submit">Submit Form</button>
```

Comment:
Can you revert this change? Since it's in the accessibility section of the guides I'm inclined to keep the current example as it's likely purposeful.

Reply:
This looks good to me @locks